gravitee-apim-console-webui/src/management/settings/shared-policy-groups/shared-policy-group/shared-policy-group-history/shared-policy-group-history.component.ts

```diff
@@ -202,7 +205,19 @@ export class SharedPolicyGroupHistoryComponent implements OnInit {
         role: 'dialog',
       })
       .afterClosed()
-      .subscribe();
+      .pipe(
+        filter((result) => result === 'RESTORE_VERSION'),
+        switchMap(() => this.sharedPolicyGroupsService.restore(sharedPolicyGroup)),
+      )
+      .subscribe({
+        next: () => {
+          this.snackBarService.success('Version has been restored. Review changes and click ‘Deploy’ to finalize the restoration.');
```

Comment:
`Version has been restored. Review changes and click ‘Deploy’ to finalize the restoration.`
should we keep the word rollback instead of restoration ?

Reply:
As in the models the term rollback is not used I used the same term `restore`. 🤷‍♂️